scripts/PauseAtHeight.py

```diff
@@ -74,6 +74,22 @@ def getSettingDataString(self):
                     "unit": "layers",
                     "type": "int",
                     "default_value": 0
+                },
+                "standby_temperature":
+                {
+                    "label": "Standby Temperature",
+                    "description": "Change the temperature during the pause",
+                    "unit": "degrees",
```

Comment:
We use "°C" as the temperature unit in Cura fdmprinter.json, so it will be nice to keep it consistent here.